### PR TITLE
fix: array to scalar deprecation warning

### LINKED ISSUE #4339: Array to scalar deprecation warning
From nightly wheels cron CI:

2023-11-03T03:16:21.1300895Z   /home/runner/work/mdanalysis/mdanalysis/package/MDAnalysis/analysis/nuclinfo.py:302: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
2023-11-03T03:16:21.1302468Z     phase_ang = (atan2(D, C) + (pi / 2.)) * 180. / pi

### PR DESCRIPTION
**Pull request by Rohitthakan**
Fixes #4339

Changes made in this Pull Request: 
This PR fixes the array to scalar deprecation warning

PR Checklist
------------
 - [X] Tests?
 - [X] Docs?
 - [X] CHANGELOG updated?
 - [X] Issue raised/referenced?

## Developers certificate of origin
- [X] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

----
📚 Documentation preview 📚: https://mdanalysis--4608.org.readthedocs.build/en/4608/

### CODE REVIEW

File: package/MDAnalysis/analysis/nuclinfo.py
Comment (RMeli) [inline package/MDAnalysis/analysis/nuclinfo.py:1]: @Rohitthakan, thank you for contributing to MDAnalysis. What is the reason for replacing the docstrings with a comment? Please revert these changes.

**Review by RMeli — CHANGES_REQUESTED** [review]

**Comment by orbeckst** [comment]
@RMeli could you please take on the role of PR shepherd? If it becomes to much, please say something and unassign yourself. Thanks!

**Comment by RMeli** [comment]
@Rohitthakan please also have a look at the tests, they are all failing.

**Comment by RMeli** [comment]
Hi @Rohitthakan, are you still working/interested in working on this?

**Comment by RMeli** [comment]
Closing for lack of replies. Please feel free to re-open if you are still working on it and want to move forward.

**Pull request closed without merging**